### PR TITLE
[Docs] PR for PITR restore should not be done when a previous restore is still in progress

### PR DESCRIPTION
**Pull request by lizayugabyte**

### CODE REVIEW

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md [hunk 1]
```diff
@@ -128,7 +130,7 @@ Create and populate a table, get a timestamp to which you'll restore, and then w
 
 ### Restore from an absolute time
 
-1. Get a timestamp.
+1. Get a timestamp:
 
     ```sh
     $ python -c 'import datetime; print datetime.datetime.now().strftime("%s%f")'
```
Comment (ddhodge) [inline docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md:136]: there's another Next, verify below

**Review by ddhodge — APPROVED** [review]
Minor comments

**Comment by ddhodge** [comment]
Should note about not running simultaneous restores be added to yb-admin reference (https://deploy-preview-13147--infallible-bardeen-164bc9.netlify.app/preview/admin/yb-admin/#restore-snapshot); maybe https://deploy-preview-13147--infallible-bardeen-164bc9.netlify.app/preview/manage/backup-restore/point-in-time-recovery/#restore-to-a-point-in-time ?

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md [hunk 2]
```diff
@@ -31,20 +31,22 @@ showAsideToc: true
...
 
-The examples on this page are deliberately simplified. In many of the scenarios presented, you could drop the index or table to recover. Consider the examples as part of an effort to undo a larger schema change, such as a database migration, which has performed several operations.
+This document contains examples that are deliberately simplified. In many of the scenarios, you could drop the index or table to recover. Consider the examples as part of an effort to undo a larger schema change, such as a database migration, which has performed several operations.
 
-{{< /tip >}}
+{{< /note >}}
 
 ## Undo data changes
 
+The process of undoing data changes involves creating and taking a snapshot of a table, and then performing a restore from either an absolute or relative time. Before attempting a restore, you need to confirm that there is no restore in progress for the subject keyspace or table: if multiple restore commands are issued, the data might enter an inconsistent state.
```
Comment (ddhodge) [inline docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md:48]: How do you confirm no restore is in progress?

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md [hunk 3]
```diff
@@ -31,20 +31,22 @@ showAsideToc: true
   </li>
 </ul>
 
-The point-in-time recovery (PITR) feature allows you to restore the state of your cluster's data (and some types of metadata) from a specific point in time. This can be relative, such as "three hours ago", or an absolute timestamp.
+The point-in-time recovery (PITR) allows you to restore the state of your cluster's data and some types of metadata from a specific point in time. This can be relative, such as "three hours ago", or an absolute timestamp.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md:34]: fixed

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md [hunk 4]
```diff
@@ -31,20 +31,22 @@ showAsideToc: true
...
 
-Refer to [Features](../../../manage/backup-restore/point-in-time-recovery/#features), [Use cases](../../../manage/backup-restore/point-in-time-recovery/#use-cases), and [Limitations](../../../manage/backup-restore/point-in-time-recovery/#limitations) for details on this feature. For more details on the `yb-admin` commands, refer to the [Backup and snapshot commands](../../../admin/yb-admin/#backup-and-snapshot-commands) section of the yb-admin documentation.
+Refer to [Features](../../../manage/backup-restore/point-in-time-recovery/#features), [Use cases](../../../manage/backup-restore/point-in-time-recovery/#use-cases), and [Limitations](../../../manage/backup-restore/point-in-time-recovery/#limitations) for details on this feature. For details on the `yb-admin` commands, refer to the [Backup and snapshot commands](../../../admin/yb-admin/#backup-and-snapshot-commands) section of the yb-admin documentation.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md:36]: Yeah, someone has changed the whole structure. Probably a lot of links are broken under the db docs.

Fixed.

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md [hunk 5]
```diff
@@ -31,20 +31,22 @@ showAsideToc: true
...
 
-You can try out the PITR feature by creating a namespace and populating it, creating a snapshot schedule, and restoring (be sure to check out the [limitations](../../../manage/backup-restore/point-in-time-recovery/#limitations)!) from that schedule.
+You can try out the PITR feature by creating a namespace and populating it, creating a snapshot schedule, and restoring from that schedule.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md:38]: fixed

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md [hunk 6]
```diff
@@ -275,11 +277,11 @@ In addition to data changes, you can also use PITR to recover from metadata chan
     (4 rows)
     ```
 
-1. Now restore back to a time before this table was altered, as in [Restore from a relative time](#restore-from-a-relative-time).
+1. Now restore back to a time before this table was altered, as described in [Restore from a relative time](#restore-from-a-relative-time).
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md:280]: fixed

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ysql.md [hunk 7]
```diff
@@ -31,20 +31,22 @@ showAsideToc: true
   </li>
 </ul>
 
-The point-in-time recovery feature allows you to restore the state of your cluster's data (and [certain types](../../../manage/backup-restore/point-in-time-recovery/#limitations) of metadata) from a specific point in time. This can be relative, such as "three hours ago", or an absolute timestamp.
+The point-in-time recovery (PITR) allows you to restore the state of your cluster's data and [certain types](../../../manage/backup-restore/point-in-time-recovery/#limitations) of metadata from a specific point in time. This can be relative, such as "three hours ago", or an absolute timestamp.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/explore/cluster-management/point-in-time-recovery-ysql.md:34]: fixed

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ysql.md [hunk 8]
```diff
@@ -31,20 +31,22 @@ showAsideToc: true
...
 
 Refer to [Features](../../../manage/backup-restore/point-in-time-recovery/#features), [Use cases](../../../manage/backup-restore/point-in-time-recovery/#use-cases), and [Limitations](../../../manage/backup-restore/point-in-time-recovery/#limitations) for details on this feature. For more details on the `yb-admin` commands, refer to the [Backup and snapshot commands](../../../admin/yb-admin/#backup-and-snapshot-commands) section of the yb-admin documentation.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/explore/cluster-management/point-in-time-recovery-ysql.md:36]: fixed

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ysql.md [hunk 9]
```diff
@@ -31,20 +31,22 @@ showAsideToc: true
...
-{{< /tip >}}
+{{< /note >}}
 
 ## Undo data changes
 
+The process of undoing data changes involves creating and taking a snapshot of a table, and then performing a restore from either an absolute or relative time. Before attempting a restore, you need to confirm that there is no restore in progress for the subject keyspace or table: if multiple restore commands are issued, the data might enter an inconsistent state.
+
 ### Create and snapshot a table
 
 Create and populate a table, look at a timestamp to which you'll restore, and then write a row.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/explore/cluster-management/point-in-time-recovery-ysql.md:52]: Yes!

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md:136]: fixed

File: docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md:48]: Fixed

**Pull request merged**